packages/calling-component-bindings/src/captionsSelector.ts

```diff
@@ -90,7 +90,7 @@ export const _captionSettingsSelector: _CaptionSettingsSelector = reselect.creat
   ) => {
     return {
       supportedCaptionLanguages: supportedCaptionLanguages ?? [],
-      currentCaptionLanguage: currentCaptionLanguage ?? '',
+      currentCaptionLanguage: currentCaptionLanguage ?? 'en',
```

Comment:
I guess english makes sense, this might be biased though? is this something thats acceptable?

Reply:
By default the calling sdk also defaults to english. So does the bot as well when starting captions.

Reply:
Down the line we can incorporate a default option? But that would be the same as giving it a spokenLanguage from the get go. Which can be easily implemented by contoso instead.